packages/design-system/src/components/WIP/ErrorState/ErrorState.stories.mdx

```diff
@@ -0,0 +1,112 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+import { FigmaImage, Use } from '~docs';
+
+import ErrorState from './ErrorState';
+
+import * as Stories from './ErrorState.stories';
+
+<Meta
+	title="[WIP] Components/ErrorState"
+	component={ErrorState}
+	parameters={{
+		status: { figma: 'wip', storybook: 'wip', react: 'wip', i18n: 'wip' },
+		figmaLink: 'https://www.figma.com/file/o5kR1lrKR9he56PyyfWaTa/Error-States',
+	}}
+/>
+
+# ErrorState
+
+Definition
+
+They describe the state where the app fails to complete an action or can't respond or do what the user wants.
+
+The scope of the error varies and could be associated with:
+
+- The entire app
+- The entire current screen
+- A specific element on the screen
+
+Depending on the scope, the error state may display directly on the screen, in dialogs, or in toasters.
+
+## Zoning
+
+<FigmaImage src="" alt="Zoning image for ErrorState" />
+
+## Style
+
+<FigmaImage src="" alt="Styles for ErrorStates" />

```

Comment:
```suggestion
<FigmaImage src="" alt="Styles for ErrorStates" />
```
@sgangnant You have no URL for this? 🤔

Reply:
Not sure what this sould be, would that be the "Specs" node ? https://www.figma.com/file/o5kR1lrKR9he56PyyfWaTa/Error-States?node-id=104%3A5159

Reply:
https://www.figma.com/file/o5kR1lrKR9he56PyyfWaTa/Error-States?node-id=104%3A5159 this node should do!